application/config.py

```diff
@@ -0,0 +1,52 @@
+import os
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config(object):
+    """ Default Cinfiguration """
+
+    DEBUG = True
+    TESTING = True
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+    SQLALCHEMY_DATABASE_URI = 'sqlite:///' + \
+        os.path.join(basedir, '/databases/app.db')
+    SQLALCHEMY_MIGRATE_REPO = os.path.join(basedir, 'db_repo')
+
+
+class Development(Config):
+    """ Development Configuration """
+
+    DEBUG = True
+    TESTING = True
+    SQLALCHEMY_TRACK_MODIFICATIONS = 'False'
+    SQLALCHEMY_DATABASE_URI = 'sqlite:///' + \
+        os.path.join(basedir, 'databases/development.db')
+    SQLALCHEMY_MIGRATE_REPO = os.path.join(basedir, 'db_repo/development')
+
+
+class Testing(Config):
+    """ Testing Configuration """
+
+    DEBUG = True
+    TESTING = True
+    SQLALCHEMY_TRACK_MODIFICATIONS = 'False'
```

Comment:
Since you have set `DEBUG` and `TESTING` variables in the Config class, you don't need to redeclare them in the `Testing` subclass since the values are the same.